metadata-ingestion/src/datahub/ingestion/api/source.py

```diff
@@ -155,10 +169,6 @@ def create(cls, config_dict: dict, ctx: PipelineContext) -> "Source":
         # can't make this method abstract.
         raise NotImplementedError('sources must implement "create"')
 
-    @abstractmethod
-    def get_workunits(self) -> Iterable[MetadataWorkUnit]:
-        pass
```

Comment:
this change doesn't look right

Reply:
Yeah, I gotta do a full pass, was a messy merge